app/Atro/Listeners/FieldManagerController.php

```diff
@@ -233,4 +237,33 @@ protected function isUniqueFieldWithoutDuplicates(string $scope, string $field):
             }
         }
     }
+
+
+    public function setEmptyValueToNullOnDisableEmptyValueActivate(Event $event){
```

Comment:
There are no reasons to do it on backend. API ca accept empty value, null value and something else.